content/pages/sprint/2023/04-hsinchu.rst

```diff
@@ -0,0 +1,77 @@
+========================================
+Scisprint 2023 April in Hsinchu
+========================================
+
+:date: 2023-03-17 17:57
+:url: sprint/2023/04-hsinchu
+:save_as: sprint/2023/04-hsinchu.html
+
+Scisprint 2023 April in Hsinchu
+========================================
+
+* Date: 15th April, Saturday, 2023
+* Time: 10:00 -- 17:00 (7 hours)
+
+To join the sprint, please bring your laptop and `sign up at kktix <#>`__.  You are also very welcome to bring your project, which
```

Comment:
Before the kktix page is made, `sign up at kktix` should not be written.  It can be `sign up (TBD)`.